Sprint-1/exercises/decimal.js

```diff
@@ -3,7 +3,13 @@ const num = 56.5678;
 // You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
 
 // Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
+let wholeNumberPart = 56;
```

Comment:
You were supposed to use a function to get the integer part of `num` (and not type the number 56)
Same for `decimalPart`, you were supposed to compute its value using an expression.
As for `roundNum`, you are in the right direction but you were suppose to apply the function to `num`, and not to 56.5.